app/src/main/res/layout/past_run.xml

```diff
@@ -10,74 +10,72 @@
 
     <ImageView
         android:id="@+id/mapImageView"
-        android:layout_width="80dp"
-        android:layout_height="80dp"
+        android:layout_width="90dp"
+        android:layout_height="90dp"
         android:background="@drawable/map_placeholder" />
 
-    <androidx.constraintlayout.widget.ConstraintLayout
+    <LinearLayout
         android:layout_width="match_parent"
-        android:layout_height="match_parent">
+        android:layout_height="match_parent"
+        android:layout_marginHorizontal="5dp"
+        android:layout_gravity="center"
+        android:orientation="vertical">
 
         <TextView
-            android:id="@+id/averageSpeedTextView"
+            android:id="@+id/dateTextView"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:layout_marginTop="36dp"
-            android:layout_marginEnd="4dp"
-            android:text="Speed:"
-            android:textColor="@color/black"
-            android:textSize="14sp"
-            app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintTop_toTopOf="parent" />
+            android:layout_marginTop="3dp"
+            android:layout_gravity="center"
+            android:text="Date"

```

Comment:
Maybe you can extract the string here (I that this string ressource already exist)

Reply:
I used this string to test the layout when I was changing it, I forgot to delete them